kratos/geometries/geometry_data.h

```diff
@@ -97,7 +97,8 @@ class GeometryData
         Kratos_Tetrahedra,
         Kratos_Hexahedra,
         Kratos_Prism,
-        Kratos_generic_family
+        Kratos_generic_family,
+        Kratos_Brep
```

Comment:
```suggestion
        Kratos_Brep,
        Kratos_generic_family
```
It is not relevant but I'd leave the generic one at the end.

Reply:
be careful, changing the order also changes the internal representation
I cannot fully remember the details but I would be careful

maybe you can google or @pooyan-dadvand @roigcarlo know better

Reply:
OK. I think this doesn't affect. But lets wait for them to merge it.

Reply:
probably won't affect and I cant find the resources I was thinking of...

All good from my side

Reply:
It affects if you use the enum as an int in a switch case with numbers:` case 5: //generic`

But not if used as an enum.

Reply:
> It affects if you use the enum as an int in a switch case with numbers:` case 5: //generic`
> 
> But not if used as an enum.

As far as I understand this would be a misuse of this so we're fine with the changes right?

Reply:
Thx thats what I meant :)

It is not a misuse in technical terms but should not be done
Afaik we don’t do it in Kratos at least I never saw it

Reply:
Thanks for the clarification! Even if someone uses it as an enum, the behavior is not changed.

Reply:
It is a misuse

Reply:
In that case, would you change the order? So that the generic one is in the end, like it was before

Reply:
yes